utils/frame/rpc/state-trie-migration-rpc/src/lib.rs

```diff
@@ -146,11 +154,14 @@ where
 
 		let hash = at.unwrap_or_else(|| self.client.info().best_hash);
 		let state = self.backend.state_at(hash).map_err(error_into_rpc_err)?;
-		let (top, child) = migration_status(&state).map_err(error_into_rpc_err)?;
+		let (top, child, total_top, total_child) =
+			migration_status(&state).map_err(error_into_rpc_err)?;
```

Comment:
nit: Would it make sense to have this function return `MigrationStatusResult` directly?